vite.config.ts

```diff
@@ -0,0 +1,12 @@
+import { vitePlugin as remix } from "@remix-run/dev";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+import { netlifyPlugin } from "@netlify/remix-edge-adapter/plugin";
+import { remixDevTools } from "remix-development-tools";
+
+
+export default defineConfig({
```

Comment:
I think you need to add the `netlifyPlugin` to the `plugins` array, it is not mentioned in there right now.